python/with-postgres/server.py

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/python3
+from railway.postgres.psycopg2 import conn
+import os
+
+cur = conn.cursor()
+try:
+    cur.execute("SELECT * from jokes")
```

Comment:
This will fail initially right? There is no `.railway` in the repo. Should the example be changed to get the current time so it will pass when someone first checks out the code.